static/app/views/settings/account/accountDetails.tsx

```diff
@@ -31,65 +39,76 @@ export type ChangeAvatarUser = Omit<
 
 const ENDPOINT = '/users/me/';
 
-interface Props extends AsyncViewProps {
-  organization: Organization;
+function makeUserEndpointQueryKey(): ApiQueryKey {
+  return [ENDPOINT];
 }
 
-class AccountDetails extends DeprecatedAsyncView<Props> {
-  getEndpoints(): ReturnType<DeprecatedAsyncView['getEndpoints']> {
-    // local state is NOT updated when the form saves
-    return [['user', ENDPOINT]];
+function AccountDetails() {
+  const organization = useOrganization();
+  const queryClient = useQueryClient();
+  const {
+    data: user,
+    isLoading,
+    isError,
+    refetch,
+  } = useApiQuery<User>(makeUserEndpointQueryKey(), {staleTime: 0});
```

Comment:
```suggestion
  } = useApiQuery<User>(['/users/me'], {staleTime: 0});
```

Reply:
yeah i'm kinda just repeating the pattern of having a function despite this not having any query parameters like others do, agree it could be a constant